sklearn/model_selection/_search.py

```diff
@@ -697,8 +712,8 @@ def fit(self, X, y=None, *, groups=None, **fit_params):
             Parameters passed to the ``fit`` method of the estimator
         """
         estimator = self.estimator
-        cv = check_cv(self.cv, y, classifier=is_classifier(estimator))
-
+        self._checked_cv_orig = check_cv(self.cv, y,
+                                         classifier=is_classifier(estimator))
```

Comment:
Are you happy with this @jnothman or would you rather call `check_cv` inside of `evaluate_candidates`?

Reply:
I think I'm okay with this.